packages/frontend/src/pages/ProjectSettings.tsx

```diff
@@ -26,22 +42,30 @@ const ProjectSettings: FC = () => {
             </div>
         );
     }
-
     return (
-        <div
-            style={{
-                display: 'flex',
-                flexDirection: 'column',
-                width: '800px',
-                margin: '20px auto',
-            }}
-        >
-            <H1 style={{ margin: 0, flex: 1 }}>
-                Edit project connection: {data.name}
-            </H1>
-            <Divider style={{ margin: '20px 0' }} />
-            <UpdateProjectConnection projectUuid={projectUuid} />
-        </div>
+        <PageBase>
```

Comment:
Much cleaner!

So now sidebars will always require routes (rather than a set of tabs)

Reply:
It is a better UX since each submenu has it's own url but I don't think it is a' 1 size fits all' situation.

Reply:
Seems good to me and will be useful for the settings page where we can link to specific areas of the settings